Sources/W3CTraceContext/SpanID.swift

```diff
@@ -16,11 +16,15 @@
 ///
 /// [W3C TraceContext: parent-id](https://www.w3.org/TR/trace-context-1/#parent-id)
 public struct SpanID: Sendable {
-    private let _bytes: Bytes
+    @usableFromInline
+    let _bytes: Bytes
```

Comment:
```suggestion
    fileprivate let _bytes: Bytes
```

Reply:
`@usableFromInline` requires `internal`. Not sure if the inlining is really necessary though 🤔 I _think_ it helps because the `withUnsafeBytes` is generic.

Reply:
It seems to help indeed.

without `@inlinable` on the `withUnsafeBytes`:

```
OTelTraceIdRatioBasedSampler
╒═══════════════════════════════╤═════════╤═════════╤═════════╤═════════╤═════════╤═════════╤═════════╤═════════╕
│ Metric                        │      p0 │     p25 │     p50 │     p75 │     p90 │     p99 │    p100 │ Samples │
╞═══════════════════════════════╪═════════╪═════════╪═════════╪═════════╪═════════╪═════════╪═════════╪═════════╡
│ Instructions *                │    1419 │    1432 │    1432 │    1432 │    1432 │    1432 │    6218 │   10000 │
├───────────────────────────────┼─────────┼─────────┼─────────┼─────────┼─────────┼─────────┼─────────┼─────────┤
│ Malloc (total) *              │       0 │       0 │       0 │       0 │       0 │       0 │       0 │   10000 │
├───────────────────────────────┼─────────┼─────────┼─────────┼─────────┼─────────┼─────────┼─────────┼─────────┤
│ Memory (resident peak) (K)    │    9339 │    9708 │    9708 │    9724 │    9748 │    9748 │    9748 │   10000 │
├───────────────────────────────┼─────────┼─────────┼─────────┼─────────┼─────────┼─────────┼─────────┼─────────┤
│ Throughput (# / s) (K)        │    1502 │    1334 │    1334 │    1334 │    1264 │     751 │     207 │   10000 │
├───────────────────────────────┼─────────┼─────────┼─────────┼─────────┼─────────┼─────────┼─────────┼─────────┤
│ Time (total CPU) (ns) *       │    3041 │    3125 │    3167 │    3209 │    3625 │    5167 │   19251 │   10000 │
├───────────────────────────────┼─────────┼─────────┼─────────┼─────────┼─────────┼─────────┼─────────┼─────────┤
│ Time (wall clock) (ns) *      │     666 │     750 │     750 │     750 │     792 │    1333 │    4833 │   10000 │
╘═══════════════════════════════╧═════════╧═════════╧═════════╧═════════╧═════════╧═════════╧═════════╧═════════╛
```


with `@inlineable` on the `withUnsafeBytes`:

```
OTelTraceIdRatioBasedSampler
╒═══════════════════════════════╤═════════╤═════════╤═════════╤═════════╤═════════╤═════════╤═════════╤═════════╕
│ Metric                        │      p0 │     p25 │     p50 │     p75 │     p90 │     p99 │    p100 │ Samples │
╞═══════════════════════════════╪═════════╪═════════╪═════════╪═════════╪═════════╪═════════╪═════════╪═════════╡
│ Instructions *                │    1321 │    1321 │    1321 │    1321 │    1321 │    1321 │    6251 │   10000 │
├───────────────────────────────┼─────────┼─────────┼─────────┼─────────┼─────────┼─────────┼─────────┼─────────┤
│ Malloc (total) *              │       0 │       0 │       0 │       0 │       0 │       0 │       0 │   10000 │
├───────────────────────────────┼─────────┼─────────┼─────────┼─────────┼─────────┼─────────┼─────────┼─────────┤
│ Memory (resident peak) (K)    │    9306 │    9675 │    9675 │    9691 │    9716 │    9716 │    9716 │   10000 │
├───────────────────────────────┼─────────┼─────────┼─────────┼─────────┼─────────┼─────────┼─────────┼─────────┤
│ Throughput (# / s) (K)        │    1502 │    1334 │    1334 │    1334 │    1334 │    1091 │      64 │   10000 │
├───────────────────────────────┼─────────┼─────────┼─────────┼─────────┼─────────┼─────────┼─────────┼─────────┤
│ Time (total CPU) (ns) *       │    3041 │    3125 │    3167 │    3167 │    3251 │    3959 │   18208 │   10000 │
├───────────────────────────────┼─────────┼─────────┼─────────┼─────────┼─────────┼─────────┼─────────┼─────────┤
│ Time (wall clock) (ns) *      │     666 │     750 │     750 │     750 │     750 │     917 │   15666 │   10000 │
╘═══════════════════════════════╧═════════╧═════════╧═════════╧═════════╧═════════╧═════════╧═════════╧═════════╛
```




Throughput P99 `751` ↗️ `1091` 🚀

Reply:
Oh, TIL. Totally fine to keep it `internal` then 👍